### PR TITLE
docs: updated file-service s3 guide

### PR DESCRIPTION
**Pull request by habibullahturkmen**
Added description for `download_file_duration` option and recently added `cache_control` option.

### CODE REVIEW

File: www/docs/content/plugins/file-service/s3.mdx [hunk 1]
```diff
@@ -121,6 +123,8 @@ Where:
 3. `<YOUR_BUCKET_REGION>` is the region code of your bucket. For example, `us-east-1`.
 4. `<YOUR_ACCESS_KEY_ID>` is the Access Key ID that you created for your user.
 5. `<YOUR_SECRET_ACCESS_KEY>` is the Secret Access Key that you created for your user.
+6. `<YOUR_CACHE_CONTROL>` is the value for caching the objects in a web browser or CDN network. For example: cache object for 10 hours, `max-age=36000`
+7. `<YOUR_DOWNLOAD_FILE_DURATION_AMOUNT>` is the expiry duration for a download url. For example: expires in 1 hour, `3600`
```
Comment (shahednasser) [inline www/docs/content/plugins/file-service/s3.mdx]:
```suggestion
7. `<YOUR_DOWNLOAD_FILE_DURATION_AMOUNT>` is the expiry duration for a download URL. For example, you can set the value to ``3600`` if you want the download URL to expire in 1 hour.
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your contribution @habibullahturkmen  ! Just a minor fix and it should be good

**Comment by habibullahturkmen** [comment]
> Thank you for your contribution @habibullahturkmen ! Just a minor fix and it should be good

Thank you so much for your feedback, I've applied your feedback.

**Review by shahednasser — APPROVED** [review]
LGTM! Thanks for your contribution!

**Review by shahednasser — COMMENTED** [review]
Can you run `yarn lint:content --fix` and commit the changes?

**Comment by habibullahturkmen** [comment]
> Can you run `yarn lint:content --fix` and commit the changes?

I've fixed the lint issue manually.

**Comment by habibullahturkmen** [comment]
> Can you run `yarn lint:content --fix` and commit the changes?

Done, thank you for your feedback.

**Comment by habibullahturkmen** [comment]
> Can you run `yarn lint:content --fix` and commit the changes?

I ran yarn lint:content --fix and pushed the code, could you please approve it. Thank you.

**Pull request merged**